content/json-ld/latest/events/event-online-url.md

```diff
@@ -0,0 +1,69 @@
+---
+---
+
+# Online url
+
+Updates the online url of an event. Only events with attendance mode `online` or `mixed` can have an online url.
+
+## HTTP request
+
+```
+PUT /events/{eventId}/online-url
+```
+Updates the online url of an event
+
+```
+DELETE /events/{eventId}/online-url
+```
+Deletes the online url of an event
+
+## Request headers
+
+| Header        | Value            | Required? |
+| ------------- | ---------------- | --------- |
+| Authorization | Bearer {token}   | true      |
+| X-Api-Key     | {apiKey}         | true      |
+| Content-Type  | application/json | false     |
+
+## Resource properties
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| eventId	| uuid | unique identifier for an event | 15277b21-f91d-42d3-8f14-a4ff66fe538a |
+
+## Request body
+
+| Property	| Type | Description |
+|--|--|--|
+| onlineUrl | url | url where the online event takes place (e.g. the link of the livestream) |
```

Comment:
Using "takes place" feels like a physical location. Not sure how to phrase it properly. Maybe something like "The url where the online event can be watched/witnessed/followed"

Reply:
I noticed you used the term "participate" inside the SAPI3 documentation, maybe that is a good term?